sepolicy_platform/file_contexts

```diff
@@ -24,6 +25,7 @@
 
 /dev/block/platform/soc/c0c4000\.sdhci/by-name/TA              u:object_r:ta_block_device:s0
 /dev/block/bootdevice/by-name/TA                               u:object_r:ta_block_device:s0
+/dev/block/mmcblk0p1                                           u:object_r:ta_block_device:s0
```

Comment:
Are you sure this is necessary? Isn't removing `/dev/block/mmcblk0p1 u:object_r:sd_block_device:s0` enough?

Reply:
Have you tried removing it ? Because TA is symlinked to this ....

Reply:
No I haven't tried it.

Reply:
I can remember with old 8994 ... Where TA was very closed about this ... So better have all TA labeled

Reply:
@luk1337 it need to be labeled or you will get denials for blk_file as block_device .....